src/main/java/com/example/notes/dto/NoteDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.notes.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@AllArgsConstructor
```

Comment:
@Setter and @Getter can be replaced with @Data for simplicity. This will also add us a @ToString annotation which will be useful for logging/debugging.